core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -743,7 +753,8 @@ class GroupCoordinator(val brokerId: Int,
                                     protocolType: String,
                                     protocols: List[(String, Array[Byte])],
                                     group: GroupMetadata,
-                                    callback: JoinCallback): MemberMetadata = {
+                                    callback: JoinCallback): Unit = {
+    val memberId = clientId + "-" + group.generateMemberIdSuffix
```

Comment:
Aha, this is why the tests are failing - left over from the merge with KIP-394 ;)